nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+
+const routerNumbers = require("./web");
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.use("/calculator", routerNumbers);
+
+//parse URL- encoded bodies
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+// router.use("/calculator", routerNumbers);
```

Comment:
```suggestion
//parse URL- encoded bodies
app.use(express.urlencoded({ extended: true }));
app.use(express.json());

app.use("/calculator", routerNumbers);

```

this will fix POST 🙂